### PR TITLE
Fea: Support RMA (Transcoding)

### PR DESCRIPTION
**Pull request by elsiosanchez**
#### Screenshot or Gif
ERROR

### CODE REVIEW

File: src/components/ADempiere/Form/VPOS2/Options/RMA/index.vue [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+
+    function exitLine(row, column, event) {
+      const { property } = column
+      if (property === 'quantity') {
+        row.isEditQty = false
+      }
+    }
+
+    function updateQuantity(quantity) {
+      console.log(quantity, line.value.product.name)
```
Comment (EdwinBetanc0urt) [inline src/components/ADempiere/Form/VPOS2/Options/RMA/index.vue]: please remove this console.log

File: src/components/ADempiere/Form/VPOS2/Options/RMA/index.vue [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+      line.value = currentRow
+    }
+
+    function deleteLine(line) {
+      line.isLoading = true
+      store.dispatch('deleteRMALine', {
+        lineId: line.id
+      })
+        .then(() => {
+          line.isLoading = false
```
Comment (EdwinBetanc0urt) [inline src/components/ADempiere/Form/VPOS2/Options/RMA/index.vue:273]: `line.isLoading = false` plase use finally statement.

File: src/components/ADempiere/Form/VPOS2/Options/RMA/previwerRMA.vue [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+
+    function exitLine(row, column, event) {
+      const { property } = column
+      if (property === 'quantity') {
+        row.isEditQty = false
+      }
+    }
+
+    function updateQuantity(quantity) {
+      console.log(quantity, line.value.product.name)
```
Comment (EdwinBetanc0urt) [inline src/components/ADempiere/Form/VPOS2/Options/RMA/previwerRMA.vue]: please remove this console.log

File: src/components/ADempiere/Form/VPOS2/Options/Shipments/index.vue [hunk 4]
```diff
@@ -155,16 +185,57 @@ export default defineComponent({
...
+
+    function exitLine(row, column, event) {
+      const { property } = column
+      if (property === 'quantity') {
+        row.isEditQty = false
+      }
+    }
+
+    function updateQuantity(quantity) {
+      console.log(quantity, line.value.product.name)
```
Comment (EdwinBetanc0urt) [inline src/components/ADempiere/Form/VPOS2/Options/Shipments/index.vue]: please remove this console.log

File: src/components/ADempiere/Form/VPOS2/Options/Shipments/lineInfo.vue [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+                <b>
+                  {{ infoLine.uom.uom.name }}
+                </b>
+              </p>
+            </el-col>
+          </el-row>
+        </el-col>
+        <el-col :span="24">
+          <el-divider>
+            {{ '1 ' + infoLine.uom.uom.name + ' (' + infoLine.uom.uom.symbol + ') ' + ' ~ ' + displayQuantity(infoLine) + ' ' + infoLine.uom.product_uom.name + ' (' + infoLine.uom.product_uom.symbol + ') ' }}
```
Comment (EdwinBetanc0urt) [inline src/components/ADempiere/Form/VPOS2/Options/Shipments/lineInfo.vue:69]: `'1 '` Is there any reason to place the static 1?

**Review by EdwinBetanc0urt — CHANGES_REQUESTED** [review]

File: src/components/ADempiere/Form/VPOS2/Options/RMA/index.vue (hunk 1 above)
Comment (elsiosanchez) [reply src/components/ADempiere/Form/VPOS2/Options/RMA/index.vue]: https://github.com/solop-develop/frontend-core/pull/1568/commits/f2c16be89c3962b250d28dd3445dd5d5a3ce039c

File: src/components/ADempiere/Form/VPOS2/Options/RMA/index.vue (hunk 2 above)
Comment (elsiosanchez) [reply src/components/ADempiere/Form/VPOS2/Options/RMA/index.vue:273]: https://github.com/elsiosanchez/frontend-core/tree/feature/feat-support-rma-transcoding

File: src/components/ADempiere/Form/VPOS2/Options/Shipments/index.vue (hunk 4 above)
Comment (elsiosanchez) [reply src/components/ADempiere/Form/VPOS2/Options/Shipments/index.vue]: https://github.com/solop-develop/frontend-core/pull/1568/commits/5b455b0b724f07ad2115182419ba4a3fa28e91ba

File: src/components/ADempiere/Form/VPOS2/Options/RMA/previwerRMA.vue (hunk 3 above)
Comment (elsiosanchez) [reply src/components/ADempiere/Form/VPOS2/Options/RMA/previwerRMA.vue]: https://github.com/solop-develop/frontend-core/pull/1568/commits/244d1e7ed4d20defe8bb8d741e3412bbdb122352

**Review by EdwinBetanc0urt — APPROVED** [review]

**Pull request merged**